src/components/questionCard/QuestionCard.tsx

```diff
@@ -0,0 +1,35 @@
+export const QuestionCard = () => {
+  return (
+    <div className="flex-col mg-s">
+      <p className="marg-un heading-text  ">
+        Q1-I am a question that is very long indeed
+      </p>
+      <div className="flex-col flex-center mg-xs">
+        <button
+          className="pd-xs play-btn outline-btn mg-xs text-center"
+          disabled={true}
+        >
+          Option One
+        </button>
```

Comment:
you can just loop over the data ie button text, instead of repeating

Reply:
yeah, i left it simple till the time i implement logic... in case it might get complicated for future.